schemas/com.snowplowanalytics.sauna.commands/envelope/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Sauna command's envelope",
+	"self": {
+		"vendor": "com.snowplowanalytics",
+		"name": "envelope",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"commandId": {
+			"type": "string",
+			"pattern": "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$"
+		},
+		"whenCreated": {
+			"type": "string"
```

Comment:
It's fine - the timestamp is in ISO8601 which can be validated in JSON Schema as:

https://github.com/snowplow/iglu-central/blob/3384a8f61a121c147c33b7ae08951ac5de9b35e0/schemas/com.urbanairship.connect/OPEN/jsonschema/1-0-0#L27